### DISCUSSION TITLE
How can I set AVIF options?

### DISCUSSION THREAD

**Discussion by dkrnl** [ImageMagick/ImageMagick#3543] [Help]
Hi!

I found only 2 AVIF options: `heic:speed` and `heic:chroma`. 

But there are more other AVIF options, for example screenshot from [squoosh.app](https://squoosh.app/):
![image](https://user-images.githubusercontent.com/734622/114577968-afa0fe80-9ca6-11eb-987c-7e541f800330.png)

Thx!

**Comment by dlemstra** [comment]
We currently only support those two options and when you set the quality to `100` the compression will be lossless.
